ImputationPipeline/AggregatePRSResults.wdl

```diff
@@ -55,8 +56,10 @@ task AggregateResults {
     Array[File] results
     Array[File] missing_sites_shifts
     String lab_batch
+    Int group_n
   }
 
+  String output_prefix = lab_batch + if group_n > 1 then  "_group_" + group_n else ""
```

Comment:
I noticed that throughout this task lab_batch is sometimes used as the WDL variable and also as an R variable, extracted from the table (line 74). Does it make sense to do a quick check in R that they match?